### ISSUE TITLE
Update v1.17.x Fails on Windows 10, Surface Pro 7

### ISSUE THREAD

**Issue by laultman** [microsoft/AzureStorageExplorer#4120]
**Storage Explorer Version**: 
**Build Number**: 
**Platform/OS**: 
**Architecture**: 
**Regression From**:  

## Bug Description ##
The installer detects a service host as running. It cannot close the service and after a couple minutes gives up. I rebooted Windows 10 and the installer still fails.

## Steps to Reproduce ##
1. Check for updates.
2. Install now.
3. Fail.

## Expected Experience ##
It should install.

## Actual Experience ##
Fails to install with error that a service is running and it cannot close the service.

**Comment by JasonYeMSFT** [comment]
Does the installer say what service is running? Usually this happens when Storage Explorer is already running.

**Comment by laultman** [comment]
It listed two services, a Microsoft service hub and another service that was not listed in the Task Manager. I cold rebooted my machine and tried to run the installer and got the same error. So I took a chance and located the service in the Task Manager and used the End Task command. Then I tried it again and it ran to completion.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by JasonYeMSFT** [comment]
@Iaultman Could you check how does those process gets started even after a reboot? There might be other applications spawning them.

**Comment by laultman** [comment]
This is a heavy duty development machine. There are all types of software installed on the machine running Visual Studio 2019. My startup list is very short. However the “restart” apps feature could be launching things that I don’t know about.

I did a cold boot in an effort to by-pass the “last known good boot” start up did not change the outcome. Interestingly, after terminating the services, nothing seemed to be affected. The only anomaly on the machine was a failed removal of a mapped Azure drive.  Windows 10 will not remove a drive letter from a machine if the Azure Drive is terminated in the Azure Portal with drives still mapped. I finally had to resort to Registry surgery to remove the drive from Windows 10. After removing the drive mapping I had a noticeable increase in speed.  My suspicion is the Windows 10 drive indexing service was at fault.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
